test/interoperability/interop_test_suite.py

```diff
@@ -0,0 +1,537 @@
+                            "altitude_hi": 400,
+                        },
+                        "time_start": time_start.strftime(DATE_FORMAT),
+                        "time_end": time_end.strftime(DATE_FORMAT),
+                    },
+                    "callbacks": {
+                        "identification_service_area_url": "https://example.com/foo"
+                    },
+                },
+            )
+            assert resp.status_code == 200, f"Failed to Insert Subscription to {dss}"
+            data = resp.json()
+            isa_ids = [isa["id"] for isa in data["service_areas"]]
+            assert (
+                self.context["isa_1_uuid"].key in isa_ids
+            ), f"{dss} did not return ISA from testStep1 when creating Subscription"
+
+    def testStep3(
+        self,
+        dss_map: Dict[str, clients.DSSClient],
+        primary_dss: str,
+        all_other_dss: List[str],
+    ) -> None:
+        """Can retrieve specific Subscription emplaced in primary DSS
+        from all DSSs."""
+        for dss in [primary_dss] + all_other_dss:
+            resp = dss_map[dss].get(
+                f"/subscriptions/{self.context['sub_1_0_uuid'].key}"
+            )
+            assert resp.status_code == 200, f"{dss} failed to get SUB_1"
+
+            data = resp.json()
+            assert (
+                self.context["sub_1_0_uuid"].key == data["subscription"]["id"]
+            ), f"{dss} did not return correct Subscription"
+
+    def testStep4(
+        self,
+        dss_map: Dict[str, clients.DSSClient],
+        primary_dss: str,
+        all_other_dss: List[str],
+    ) -> None:
+        """Can query all Subscriptions in area from all DSSs."""
+        all_dss = [primary_dss] + all_other_dss
+        all_sub_1 = set()
+        for index in range(len(all_dss)):
+            all_sub_1.add(self.context[f"sub_1_{index}_uuid"].key)
+        for index, dss in enumerate(all_dss):
+            resp = dss_map[dss].get(f"/subscriptions?area={GEO_POLYGON_STRING}")
+            assert resp.status_code == 200, f"{dss} failed to get SUB_1 by area"
+
+            returned_subs = set([x["id"] for x in resp.json()["subscriptions"]])
+
+            missing_subs = all_sub_1 - returned_subs
+            assert (
+                missing_subs == set()
+            ), f"{dss} returned too few Subscriptions, missing: {missing_subs}"
+
+            extra_subs = returned_subs - all_sub_1
+            assert (
+                extra_subs == set()
+            ), f"{dss} returned too many Subscriptsions, extra: {extra_subs}"
+
+    def testStep5(
+        self, dss_map: Dict[str, clients.DSSClient], primary_dss: str, **kwargs
+    ) -> None:
+        """Can modify ISA in primary DSS, ISA modification triggers 
+        subscription notification requests"""
+        isa_1_uuid = self.context["isa_1_uuid"].key
+        resp = dss_map[primary_dss].get(f"/identification_service_areas/{isa_1_uuid}")
```

Comment:
We should retain `version` from a previous call; a compliant DSS is not required to implement this endpoint.

Reply:
Really? I thought it was required based on this https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/BenjaminPelletier/DiscoveryAndSynchronization/rid/api/dss.yaml#tag/dss/paths/~1v1~1dss~1identification_service_areas~1{id}/get